styles.css

```diff
@@ -0,0 +1,209 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+}
+
+body {
+  background-color: #f4f5f6;
+  font-family: 'Slabo 27px', serif;
+}
+
+.container {
+  background-color: white;
+  max-width: 1310px;
+  padding-left: 0;
+  padding-right: 0;
+  overflow-x: hidden;
+}
+
+.navbar {
+  width: 100%;
+}
+
+.bottom-nav {
+  border-bottom: 2px solid #f4f5f6;
+}
+
+.category {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-2 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-3 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 9.2em;
+}
+
+.category-4 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 10.7em;
+}
+
+.category-5 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.category-6 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 5.9em;
+}
+
+.category-7 {
+  font-family: 'Roboto Condensed', sans-serif;
+  bottom: 12.2em;
+}
+
+.vs-img {
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  bottom: -20px;
+  z-index: 1;
+}
+
+.vs-line {
+  margin-bottom: -5px;
+  margin-top: -15px;
+}
+
+.features2 img {
+  width: auto;
+}
+
+.the-end {
+  position: relative;
+  left: -120px;
+  top: 18px;
+}
+
+.in-depth {
+  margin-left: -6rem !important;
+}
+
+.text-position {
+  margin-left: -1.1rem !important;
+}
+
+.skew {
+  position: relative;
+  top: 4.3rem;
+  right: -1em;
+  width: 60px;
+  height: 25px;
+  clip-path: polygon(0 0, 100% 0%, 100% 100%, 20% 100%);
+}
+
+@media only screen and (max-width: 767px) {
+  .sub-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .footer-info-top {
+    flex-wrap: wrap;
+  }
+
+  .sub-top-btn {
+    font-size: 10px;
+  }
+
+  .category {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 12.2em;
+  }
+
+  .category-1 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 12.2em;
+  }
+
+  .category-2 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 7.7em;
+  }
+
+  .category-3 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-5 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-6 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 5.6em;
+  }
+
+  .category-7 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+
+  .category-8 {
+    font-family: 'Roboto Condensed', sans-serif;
+    bottom: 6.2em;
+  }
+}
```

Comment:
No repeating code, you can notice all this classes are using the same font-family and most of them the same font-size, please refactor that.